lib/tagging/linkables.rb

```diff
@@ -0,0 +1,43 @@
+# Linkables
```

Comment:
Reuse. ♻️  I'm just concerned we have a lot of repeated code in the various publishing apps that will never get cleaned up, and it all adds maintenance overhead.